pennylane/ops/op_math/exp.py

```diff
@@ -170,6 +170,8 @@ def _queue_category(self):
     # pylint: disable=arguments-renamed, invalid-overridden-method
     @property
     def has_decomposition(self):
+        from pennylane.pauli.utils import is_pauli_word  # pylint:disable=import-outside-toplevel
+
         return math.real(self.coeff) == 0 and is_pauli_word(self.base)
```

Comment:
I agree with @soranjh 
```suggestion
        return math.real(self.coeff) == 0 and qml.pauli.is_pauli_word(self.base)
```